roles/ovirt-provider-ovn-driver/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- block:
+  - name: Install ovirt-provider-ovn-driver
+    yum:
```

Comment:
Removing my ack until resolved the inclusion those packages on NGN and install by default, more details at https://bugzilla.redhat.com/show_bug.cgi?id=1488064#c9

Reply:
OK, based on bugzilla discussion, removing my nack